### PR TITLE
11 feature 404 not found page

### PR DESCRIPTION
**Pull request by NDenheim**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -11,6 +12,7 @@ const App = () => {
         <Route path="/events" element={""} />
```
Comment (Charlie-robin) [inline src/App.tsx:12]: I think we will need to change the `/home` endpoint for now to `/` as on the [pre-deploy-site](https://union-client-project--pr9-11-feature-404-not-f-lzjhsx3p.web.app/) the 404 page is being shown

File: src/pages/Error/Error.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import "./Error.scss";
+import ErrorImage from "../../assets/images/404Error.png";
```
Comment (Charlie-robin) [inline src/pages/Error/Error.tsx:2]: This is not a component so should not be PascalCased the file should be lowercase hyphenated as well. 

`import errorImage from "../../assets/images/404-error.png";`

**Review by Charlie-robin — CHANGES_REQUESTED** [review]
Some small things to tweak. 

Could you also add some detail to readme about your page please.

**Comment by Charlie-robin** [comment]
@sabifromtherock @NDenheim This PR now has conflicts, can you please resolve them before we review the changes

**Review by Charlie-robin — APPROVED** [review]

**Review by olirob93 — APPROVED** [review]
lovely

**Pull request merged**